site-icon-maskable/site-icon-maskable-block-editor.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Handles the 'site_icon_maskable' setting within a full-site-editing context
+ *
+ * @package PWA
+ */
+
+namespace PWA_WP;
+
+use Error;
+
+use function add_action;
+use function plugins_url;
+use function register_setting;
+use function wp_enqueue_script;
+use function wp_set_script_translations;
+
+/**
+ * [pwa__enqueue_block_editor_assets description]
+ *
+ * @package PWA
+ * @since   0.8.0-alpha
+ * @throws Error Fatals out when the block-filter files weren't built.
+ *
+ * @see  https://developer.wordpress.org/reference/hooks/enqueue_block_editor_assets/
+ */
+function enqueue_block_editor_assets__site_icon_maskable() {
+	$dir = __DIR__;
+
+	$script_asset_path = "$dir/../build/site-icon-maskable.asset.php";
+	if ( ! file_exists( $script_asset_path ) ) {
+		throw new Error(
+			'You need to run `npm run blocks:start` or `npm run blocks:build` first, for the "core/site-logo"-block filter to work.'
+		);
```

Comment:
Since `Error` this is PHP 7+, I think this should be reworked to be be compatible with PHP 5.6 since that's still the minimum version supported.

Reply:
- [X] Remove `throw new error` and instead make use of `_pwa_print_build_needed_notice` instead, when files were not built https://github.com/GoogleChromeLabs/pwa-wp/pull/919/commits/47370814503dab51d4ddbcd45862479ca2ce9db6